### ISSUE TITLE
contrib/99designs/gqlgen: bump version in go.mod

### ISSUE THREAD

**Issue by nathan-genies** [DataDog/dd-trace-go#1452]
Hello,

We're trying to leverage the tracing library for 99designs but looks like the current code is not compatible with the latest 99design release. Could you take a look and maybe use vendor/ to provide a stable release? Thanks!

```
go mod tidy
go: finding module for package gopkg.in/DataDog/dd-trace-go.v1/contrib/99designs/gqlgen
go: found gopkg.in/DataDog/dd-trace-go.v1/contrib/99designs/gqlgen in gopkg.in/DataDog/dd-trace-go.v1 v1.41.0
go: finding module for package github.com/99designs/gqlgen/example/todo
go: found github.com/99designs/gqlgen/example/todo in github.com/99designs/gqlgen/example v0.0.0-20220208183958-9262b35865e4
go: github.com/99designs/gqlgen/example@v0.0.0-20220208183958-9262b35865e4 requires
        github.com/99designs/gqlgen@v0.0.0-00010101000000-000000000000: invalid version: unknown revision 000000000000
make: *** [vendor] Error 1
```

**Comment by david-alza** [comment]
Just jumping in to say we're facing the same thing when trying to add this (we're on gqlgen v0.17.16). 

I wonder if there was a specific reason to integrate this originally against v0.14? https://github.com/DataDog/dd-trace-go/commit/39187837f89f4f2b8e21ec9660b50ff9231208f4#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R7

**Comment by katiehockman** [comment]
Thanks for filing an issue and sending a PR 👍

**Comment by ajgajg1134** [comment]
Resolved in #1457
